find_similar/tokenize.py

```diff
@@ -21,14 +21,21 @@
     'vz',
     # 'мя' # 2-мя, 3-мя, ...
 }
-try:
-    stopwords_with_language = stopwords.words('russian')
-except LookupError:
-    nltk.download('stopwords')
-    nltk.download('punkt')
-    stopwords_with_language = stopwords.words('russian')
-STOP_WORDS = set(stopwords_with_language).union(PUNCTUATION_SET)
-STOP_WORDS = STOP_WORDS.union(UNUSEFUL_WORDS)
+
+STOP_WORDS_NO_LANGUAGE = PUNCTUATION_SET.union(UNUSEFUL_WORDS)
+
+
+def add_nltk_stopwords(stop_words: set, language: str):
+    try:
+        stopwords_with_language = stopwords.words(language)
+    except LookupError:
+        nltk.download('stopwords')
+        nltk.download('punkt')
+        stopwords_with_language = stopwords.words(language)
+    except OSError:
+        stopwords_with_language = stopwords.words('russian')
```

Comment:
OSError appears when nltk module is installed correctly but there is a typing error in language string, i. e. 'rusian' or 'inglish' or just random set of chars, or nltk module does not any language. So it is being replaced by default 'russian'

Reply:
Understand. I think it's wrong path. Because "explicit is better than implicit" and if the language is wrong it's Exception and we should raise one. I propose to create our own Exception what language is wrong, or to find existing (may be in nltk or elsewhere)

Reply:
Made special exception for this case